libcnb/src/runtime.rs

```diff
@@ -65,7 +65,7 @@ pub fn libcnb_runtime<B: Buildpack>(buildpack: &B) {
                     eprintln!(
                         "https://github.com/buildpacks/spec/blob/main/buildpack.md#detection"
                     );
-                    exit(1);
+                    exit(exit_code::GENERIC_UNSPECIFIED_ERROR);
```

Comment:
Should these (the "Usage" errors) get their own exit code, given the only reason they would occur is due to a libcnb issue (eg incorrect packaging of the buildpack or incorrect args handling for a new future buildpack API version), or else an end user trying to run `bin/build --help` or something?

That is, these failure modes seem similar to the other internal ones that currently have their own exit code too - so I'm unsure as to why these are inconsistent?

Reply:
Probably inconsistent due to being introduced at different times and maybe different authors? I agree that exit codes are a bit messy right now. At least that surfaces now with the constants. :)